src/slack.js

```diff
@@ -209,11 +230,17 @@ export async function messageReceived(msg) {
       message: {
         text: messageText,
         ts: timestamp,
+        subtype,
         user,
       },
       channel,
     } = msg;
 
+    if (subtype === 'tombstone') { // message is deleted
+      removeUnknownOrder(timestamp);
```

Comment:
I get `Unhandled rejection Error: no_reaction` when there are already other responses in the thread.